### DISCUSSION TITLE
Is there any way to mention a user in description or in comments?

### DISCUSSION THREAD

**Discussion by naregboynukalin** [pycontribs/jira#1367] [Q&A]
Hello, I'm trying to mention people while updating the description of an issue with jira python but I have a problem.

```
issue = jira.issue("<issue_key>")
issue.update(fields={'description': "[~<username>]"})
```
It's trying to mention the user with [~<username>] convention but even if I tried to use accountId, username or display name, it shows only "User" and when I click on it, it displays an error like below:

<img width="369" alt="image" src="https://user-images.githubusercontent.com/43938095/165102123-e9d61c31-e820-474e-ad33-3fc8f9ecbe2a.png">

Do you know any way to fix it?

**Comment by adehad** [comment]
I believe the right format: `f"[~accountId:{your_user_accountId}]"`, but might be worth checking an issue where there is a user already in the description and see what is shown

**Reply by studioj** [reply]
```
username = "studioj"
string_to_comment = f"[~{username}] added this comment"
jira.add_comment("<issue_key>", string_to_comment)
```
this works for comments on Jira Server
